build_tools/packaging/python/templates/rocm/src/rocm_sdk/__main__.py

```diff
@@ -34,6 +34,19 @@ def _do_path(args: argparse.Namespace):
         sys.exit(1)
 
 
+def _do_init(args: argparse.Namespace):
+    from . import _devel
+
+    try:
+        # The function `_devel.get_devel_root()` calls into `_expand_devel_contents`
+        # if contetens for development were not yet unpacked.
+        root_path = _devel.get_devel_root()
+    except ModuleNotFoundError as e:
+        print(f"ERROR: {e}", file=sys.stderr)
+        sys.exit(1)
+    print(f"Devel contents expanded to '{root_path}'.")
```

Comment:
nit: Maybe omit the `.` from this log line? Seems unnecessary
```suggestion
    print(f"Devel contents expanded to '{root_path}'")
```